client/components/signup-form/index.jsx

```diff
@@ -38,9 +38,11 @@ import { mergeFormWithValue } from 'signup/utils';
 import SocialSignupForm from './social';
 import { recordTracksEventWithClientId as recordTracksEvent } from 'state/analytics/actions';
 import { createSocialUserFailed } from 'state/login/actions';
+import { getCurrentOAuth2Client } from 'state/ui/oauth2-clients/selectors';
 import Card from 'components/card';
 import { preventWidows } from 'lib/formatting';
 
+
```

Comment:
Lint error:
` 44:1  error  More than 1 blank line not allowed  no-multiple-empty-lines`